pkg/runtime/runtime_test.go

```diff
@@ -2087,7 +2099,7 @@ func TestInitBindings(t *testing.T) {
 		c.ObjectMeta.Name = "testInputBinding"
 		c.Spec.Type = "bindings.testInputBinding"
 		err := r.initBinding(c)
-		assert.NotEqual(t, "couldn't find input binding bindings.testInputBinding", err.Error())
```

Comment:
The error here was actually "app channel not initialized" which is not checked in this method anymore